### DISCUSSION TITLE
Congratulations and Recommendations

### DISCUSSION THREAD

**Discussion by ricardo-hl** [jonataslaw/getx#2331] [Ideas]
**1.** The fact that you do not have more collaborators leaves a lot to think about, especially because a single person is unable to respond quickly to all the bugs and the insecurity that the project can be stopped.
**2.** The fact that you have everything in one place also gives a lot of insecurity, it would be better if it were modular. Example: GetX Core, GetX Route, GetX DI, GetX etc...

**If you take these two recommendations, surely GetX can be the king, for now flutter_bloc, provider, riverpod are winning the market.**

God Job

**Comment by SirBarksALot** [comment]
1. Winning the market? 10k likes vs 13k !!!combined!!!
2. That is the beauty of this project. Get is not just a package, it enhance all parts of flutter. Treat it equally as
```yaml
  flutter:
    sdk: flutter
```
